### PR TITLE
Image repositories are not disabled by yum anymore

### PR DESCRIPTION
**Pull request by dbecvarik**
Concreate was chaged in a way, that existing repositories in image
are not disabled during build phase. This enables easy community
images development.

### CODE REVIEW

**Comment by dbecvarik** [comment]
Do not merge, I found a bug I need to fix during community image streams builds.

**Comment by dbecvarik** [comment]
this should be ready to merge now

File: concreate/templates/template.jinja [hunk 1]
```diff
@@ -47,19 +47,19 @@ EXPOSE {%- for port in helper.ports(ports) %} {{ port }}{% endfor %}
 USER root
 
 {% if packages and packages.install %}
-{% if additional_repos %}
+{% if packages.repositories %}
 # Add custom repo files
 COPY repos/*.repo /etc/yum.repos.d/
```
Comment (goldmann) [inline concreate/templates/template.jinja]: We probably should add selectively files here, just like we do it at removal time, so nothing we do not expect will not be added to container and then NOT removed.

Comment (dbecvarik) [reply concreate/templates/template.jinja]: good catch, thanks

**Pull request merged**